superset/views/core.py

```diff
@@ -2288,7 +2289,7 @@ def csv(self, client_id):
             csv = df.to_csv(index=False, **config.get('CSV_EXPORT'))
         response = Response(csv, mimetype='text/csv')
         response.headers['Content-Disposition'] = (
-            'attachment; filename={}.csv'.format(query.name))
+            'attachment; filename={}.csv'.format(unidecode(query.name)))
```

Comment:
Maybe this is enough?

```
query.name.encode('ascii', 'ignore')
```

Reply:
For a German there is a big difference, whether the Umlauts are entirely dropped or just replaced by their closest ascii counterpart.

I do understand that it may seem inappropriate to introduce yet another dependency for such a minor change, but I feel like this is affecting the user experience, especially of the less technically versed users.

You may safely skip the following pleadings…

----------------------------------------------

There are languages, where this is even more relevant. To name an extreme example, in Turkish there exists this nice word `düsündürttürücülügümüzün`. Personally I think that `dusundurtturuculugumuzun` may still be readable, but `dsndrttrclgmzn` is not. 

Also think of the Finnish people: A SQLLab tab on the unemployment in the town of Järvenpää `työttömyys_Järvenpää` could either be `tyttmyys_Jrvenp` or `tyottomyys_Jarvenpaa`.

And finally, also our Frensh speaking friends may benefit from not dropping the letters: ` élevé` becomes `lev` or `eleve`.

----------------------------------------------